include/cglm/vec3.h

```diff
@@ -793,6 +796,59 @@ glm_vec3_lerp(vec3 from, vec3 to, float t, vec3 dest) {
   glm_vec3_add(from, v, dest);
 }
 
+/*!
+ * @brief threshold function (according to OpenCL specs)
+ *
+ * @param[in]   edge    threshold
+ * @param[in]   x       value to test against threshold
+ * @param[out]  dest    destination
+ */
+CGLM_INLINE
+void
+glm_vec3_step(float edge, vec3 x, vec3 dest) {
+  dest[0] = glm_step(edge, x[0]);
```

Comment:
https://thebookofshaders.com/glossary/?search=step

You only provided edge as single dimension. it's size could be same as vector? We could provide `glm_vec3_step_uni` or `glm_vec3_step1`... for single dimension. Or `glm_vec3_stepv3` or `glm_vec3_step3` could be used for vec3?

Reply:
Yes, what you are saying is true.
I will change the single dimension case to glm_vec3_step_uni() and I will add vector versions.
Same goes for smoothstep.

Reply:
I think `x` must be first parameter, what you you think?